ocaml/xapi/xapi_pusb.ml

```diff
@@ -31,10 +31,12 @@ let create ~__context ~uSB_group ~host ~other_config ~path
 
 let scan_start ~__context usbs =
   let host = Helpers.get_localhost ~__context in
-  let known_usb =
+  let local_pusb_records =
```

Comment:
We now have 
* `local_pusb_records`
* `local_usb`
* `get_local_usb`
in this code and I think this is confusing. Can we find better names or avoid defining them all?

Reply:
Thinking about it: maybe it's hard to avoid. I would like to suggest something else, though: `known_usb` should be become `known_local_usb`. Overall this commit is about making sure we only act on local usb devices and the error was that we operated on all usb devices in the pool.

Reply:
Maybe `known_local_usbs` then (in plural). What what does "known" actually mean? Is it that ones that are in the DB? Then how about `local_psubs_in_db`?

Reply:
Yes , "known" means in DB

Reply:
The names of the (existing) functions `get_known_usb` and `get_local_usb` are a bit misleading as well, since they take a list, then map it to some other data structure, and turn the outcome into a set.

Reply:
get_local_usb actually takes a json format data and get_known_usb takes a list

Reply:
Even more confusing then 😂

Reply:
confusing more... :)

Reply:
I think change konw_usb to known_usb_set and local_usb to local_usb_set will make this less confuse

Reply:
local_pusb_records should be known_pusbs_in_db. How about these names?

Reply:
I think "local" is the one thing that _is_ clear, actually :) We always use that for "local to the host".

Reply:
@robhoes  @lindig , please help to review the updates. Not sure now the name still confusing.